src/java/org/apache/cassandra/schema/CompressionParams.java

```diff
@@ -40,10 +42,14 @@
 import org.apache.cassandra.io.util.DataOutputPlus;
 import org.apache.cassandra.net.MessagingService;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import static java.lang.String.format;
 
 public final class CompressionParams
 {
+    private static final Logger logger = LoggerFactory.getLogger(CompressionParams.class);
```

Comment:
if you log in the implementation as suggested then this might go away (with imports too)